docs/src/components/home/HomeEditor.tsx

```diff
@@ -103,4 +103,6 @@ const HomeEditor = () => {
   );
 };
 
-export default HomeEditor;
+HomeEditor.displayName = 'HomeEditor';
+
+export default React.memo(HomeEditor);
```

Comment:
Think there's a chance that this can be recreated since `React.memo` is returning a new function
```suggestion
const MemoHomeEditor = React.memo(HomeEditor);
export default MemoHomeEditor;
```